src/validationinterface.cpp

```diff
@@ -68,6 +68,16 @@ void UnregisterAllValidationInterfaces() {
     g_signals.UpdatedBlockTip.disconnect_all_slots();
 }
 
+size_t RecursiveDynamicUsage(
+    std::vector<BatchScanner*> &batchScanners)
+{
+    size_t usage = 0;
+    for (auto& batchScanner : batchScanners) {
+        usage += batchScanner->RecursiveDynamicUsage();
+    }
+    return usage;
```

Comment:
Lol I was about to suggest `batchScanners.map(|scanner| scanner->RecursiveDynamicUsage()).sum()`. This is my brain on some unholy C++/Rust hybrid :-p